common/containers/Tabs/ScheduleTransaction/components/Fields/TimeBounty/TimeBountyField.tsx

```diff
@@ -1,41 +1,147 @@
-              <div className="ScheduleFields-field-title-tooltip">
-                <Tooltip>{translateRaw('SCHEDULE_TIMEBOUNTY_TOOLTIP')}</Tooltip>
-                <Help className="ScheduleFields-field-title-help" />
+type Props = OwnProps & DispatchProps & StateProps;
+
+interface State {
+  advanced: boolean;
+}
+
+class TimeBountyFieldClass extends Component<Props, State> {
+  public state = {
+    advanced: false
+  };
+
+  public render() {
+    const { currentTimeBounty, isReadOnly, isValid } = this.props;
+    const { advanced } = this.state;
+
+    return (
+      <>
+        <div className="input-group-wrapper">
+          <label className="input-group">
+            <div className="input-group-header">
+              <span className="ScheduleFields-field-title">
+                <div className="ScheduleFields-field-title-text">
+                  {translate('SCHEDULE_TIMEBOUNTY')}
+                </div>
+                <div className="ScheduleFields-field-title-tooltip">
+                  <Tooltip>{translateRaw('SCHEDULE_TIMEBOUNTY_TOOLTIP')}</Tooltip>
+                  <Help className="ScheduleFields-field-title-help" />
+                </div>
+              </span>
+            </div>
+            <div className="TimeBountyField">
+              <div className="TimeBountyField-wrapper">
+                {advanced ? (
+                  <Input
+                    isValid={isValid}
+                    className="TimeBountyField-input"
+                    type="text"
+                    value={currentTimeBounty.raw}
+                    placeholder={translateRaw('X_CUSTOM')}
+                    readOnly={isReadOnly}
+                    spellCheck={false}
+                    onChange={this.handleOnChange}
+                    showInvalidWithoutValue={true}
+                  />
+                ) : (
+                  <GroupedRadioToggle
+                    options={EAC_SCHEDULING_CONFIG.PRESET_TIME_BOUNTIES}
+                    isValid={isValid}
+                    onChangeHandler={this.handleOnChange}
+                    selectedValue={currentTimeBounty.raw}
+                  />
+                )}
+
+                <div className="TimeBountyField-preview">
+                  <span className="TimeBountyField-preview-display">
+                    ${this.getUSDEstimation(currentTimeBounty)}
+                  </span>
+                </div>
               </div>
-            </span>
-          </div>
-          <Input
-            isValid={isValid}
-            type="text"
-            value={currentTimeBounty.raw}
-            placeholder={translateRaw('SCHEDULE_TIMEBOUNTY_PLACEHOLDER')}
-            readOnly={!!(isReadOnly || readOnly)}
-            spellCheck={false}
-            onChange={onChange}
-          />
-        </label>
-      </div>
-    )}
-  />
-);
+            </div>
+          </label>
+        </div>
+        <a
+          href="#"
+          onClick={() => this.setState({ advanced: !advanced })}
```

Comment:
@Bagaric 
Let's switch this anonymous function for `onClick={this.toggleAdvanced}` and add
```
  private toggleAdvanced = () => this.setState(prevState => ({
    advanced: !prevState.advanced
  }));
```